CONTRIBUTING.md

```diff
@@ -191,3 +191,41 @@ these steps:
 >    ```
 >    The error is related to the `tcl` package. You should uninstall the `tcl`
 > package to resolve the error.
+
+## Error handler
+
+Use the `DPCTL_VERBOSITY` environment variable to show errors of different severity levels of the C-API in the console.
+You can specify one of the following severity levels (in increasing order of severity): `warning` and `error`.
+
+```bash
+export DPCTL_VERBOSITY=error
+```
+
+Messages of a given severity are shown not only in the console for that severity, but also for a lower severity.For example, the severity level `error` will output severity errors for `error` and `warning` to the console.
+
+## Google logging libriry (glog)
+
+The dpctl error handler provides logging for exceptions of different severity levels in `libsyclinterface` using the [glog](https://github.com/google/glog) library.
+To use logging during development , follow these steps:
+
```

Comment:
Please edit the two lines to:

"Dpctl's error handler for `libsyclinterface` can be optionally configured to use [glog](https://github.com/google/glog). To use glog, follow these steps: